### ISSUE TITLE
nggettext_extract silently drops non *.html input

### ISSUE THREAD

**Issue by leamas** [rubenv/angular-gettext#271]
New to this, possibly making all kind of errors. That said, it seems that the nggettext_extract  grunt task silenlty drops any input pattern not ending in 'html'. E. g. , when using something like

                files: {
                      // 'po/template.pot': ['foo.cshtml']
                      'po/template.pot': ['foo.html']
                }
 
_foo.html_ works  whereas _foo.cshtml_  silently fails.

Not accepting non-html files becomes a problem in any templating system, some of which are used together with angular. I understand that changing this behaviour might break some use cases, so there might be some  specific configuration involved in accepting all kinds of input files.

Or, perhaps, there already is such an option, I just don't find it? The need seems obvious to me (?)

On top of this, the task completing without a single translation should IMHO generate a warning.

**Comment by cawa-93** [comment]
I completely agree. When you try to extract * .tpl files (which is a valid html code), nothing happens. Translations were not found. At the same time there has been no reports of missing files.
